package.json

```diff
@@ -53,6 +53,10 @@
     "@npmcli/template-oss": "file:./",
     "tap": "^16.0.0"
   },
+  "peerDependencies": {
```

Comment:
We really don't want this installed all the time, only for this PR

Reply:
yeah, this PR was an experiment to see if it passed. my goal was to have a way to version both those dependencies in all the repos that consume template-oss. currently this action could break if either of those deps have major versions with breaking changes